src/shared/components/header/index.js

```diff
@@ -1,18 +1,63 @@
-import React from 'react';
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
 import { PropTypes } from 'prop-types';
 import classNames from 'classnames';
+import LocalesBar from 'shared/components/locales-bar';
 import DesktopNavbar from 'shared/components/navbar/desktop';
 import MobileNavbar from 'shared/components/navbar/mobile';
 // import Banner from 'shared/components/banner';
 import styles from './index.module.css';
 
-const Header = ({ className }) => (
-    <header className={ classNames(styles.header, className) }>
-        {/* <Banner /> */}
-        <DesktopNavbar />
-        <MobileNavbar />
-    </header>
-);
+class Header extends Component {
+    constructor() {
+        super();
+
+        this.state = {
+            scrolled: false,
+        };
+    }
+
+    componentDidMount() {
+        window.addEventListener('scroll', this.handleScroll);
```

Comment:
Did you consider using [IntersectionObserver](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API)? 

You will be listening to the scroll events all the time this way,

Reply:
I didn't consider IntersectionObserver. But I'll create a card to implement it later.